examples/multiple-dbs/main.tf

```diff
@@ -8,116 +8,124 @@ data "aws_region" "current" {}
 #####
 # VPC and subnets
 #####
-module "vpc" {
-  source  = "terraform-aws-modules/vpc/aws"
-  version = "~> 2.62"
+data "aws_vpc" "default" {
+  default = true
+}
 
-  name = "simple-vpc"
+data "aws_subnet_ids" "all" {
```

Comment:
source only public ones

Reply:
It's only using the subnets within the default vpc. I did a quick check by outputting the values and confirmed it's only 3 subnets from our default vpc.